packages/engine/errors/1009.md

```diff
@@ -0,0 +1,6 @@
+---
+original: "Trailing comma not allowed."
+excerpt: "You've added a trailing comma when you're not supposed to add it"
+---
+
+You should remove the trailing comma where the typescript compiler is pointing to make Typescript happy :)
```

Comment:
One thing I've noticed is that it's best to write these errors where TypeScript is the 'I' - i.e. 'I'm pointing here, there's an excess comma there.' Picked this up from Elm, which does it beautifully. Makes it a lot more conversational. We should add this to the contrib guide.

Reply:
Yeah, agree.